doc/getting-started-guide/faq.rst

```diff
@@ -186,6 +186,136 @@ What other projects leverage xarray?
+
+.. csv-table::
+   :header: "File Format", "xarray IO Function", " Other Packages"
+   :widths: 15, 35, 15
+
+   "NetCDF (.nc, .nc4, .cdf)","xarray.open_dataset() OR xarray.open_mfdataset()", "`netCDF4 <https://pypi.org/project/netCDF4/>`_, `netcdf <https://pypi.org/project/netcdf/>`_ , `cdms2 <https://cdms.readthedocs.io/en/latest/cdms2.html>`_"
+   "HDF5 (.h5, .hdf5)","xarray.open_dataset() OR xarray.open_mfdataset()", "`h5py <https://www.h5py.org/>`_, `pytables <https://www.pytables.org/>`_ "
+   "GRIB1/GRIB2 (.grb, .grib)", "xarray.open_dataset()", "`cfgrib <https://pypi.org/project/cfgrib/>`_, `pygrib <https://pypi.org/project/pygrib/>`_"
+   "Zarr","xarray.open_zarr()","`zarr <https://zarr.readthedocs.io/en/stable/>`_ , `fsspec <https://filesystem-spec.readthedocs.io/en/latest/>`_"
+   "CSV (.csv)","xarray.open_dataset()<br>xarray.open_mfdataset()","`pandas <https://pandas.pydata.org/>`_ , `dask <https://www.dask.org/>`_ "
+   "Excel (.xls, .xlsx)","xarray.open_dataset()","`pandas <https://pandas.pydata.org/>`_, `openpyxl <https://pypi.org/project/openpyxl/>`_ "
+   "JSON (.json)","xarray.open_dataset()","`json <https://docs.python.org/3/library/json.html>`_, `pandas <https://pandas.pydata.org/>`_"
+
+To use these IO functions in xarray, you can simply call them with the path to the file(s) you want to read as an argument.
+
+NetCDF
+------
+::
+
+  import xarray as xr
+
+  # Open a NetCDF file using xarray
+  ds = xr.open_dataset('/path/to/my/file.nc')
+
+  # Open multiple NetCDF files as a single dataset using xarray
+  ds = xr.open_mfdataset('/path/to/my/files/*.nc')
+
+  # Open a NetCDF file using netCDF4 package
+  from netCDF4 import Dataset
+  nc = Dataset('/path/to/my/file.nc', 'r')
+
+  # Open a NetCDF file using cdms2 package
+  import cdms2
+  f = cdms2.open('/path/to/my/file.nc')
+
+HDF5
+----
+::
+
+  import xarray as xr
+
+  # Open a HDF5 file using xarray
+  ds = xr.open_dataset('/path/to/my/file.h5')
+
+  # Open a HDF5 file using h5py package
+  import h5py
+  f = h5py.File('/path/to/my/file.h5', 'r')
+
+  # Open a HDF5 file using pytables package
+  import tables
+  f = tables.open_file('/path/to/my/file.h5', 'r')
+
+GRIB1/GRIB2
+-----------
+::
+
+  import xarray as xr
+
+  # Open a GRIB file using xarray
+  ds = xr.open_dataset('/path/to/my/file.grb')
+
+  # Open a GRIB file using cfgrib package
+  import cfgrib
+  ds = cfgrib.open_dataset('/path/to/my/file.grb')
+
+  # Open a GRIB file using pygrib package
+  import pygrib
+  grbs = pygrib.open('/path/to/my/file.grb')
+
+
+Zarr
+----
+::
+
+  import xarray as xr
+
+  # Open a Zarr store using xarray
+  store = xr.open_zarr('/path/to/my/store.zarr')
+  ds = xr.open_zarr(store)
+
+  # Open a Zarr store using zarr package
+  import zarr
+  store = zarr.open('/path/to/my/store.zarr')
```

Comment:
I think we generally only want to list other methods of opening certain file formats if those methods return xarray objects (i.e. an `xarray.Dataset`). No-where else in xarray is the reader really going to use the `zarr.store` object returned here, the FAQ should be about "how do I open X file format as an xarray.Dataset?"